corio.py

```diff
@@ -274,10 +242,7 @@ def main(options):
                                         tests_details=tests_to_execute, aborted=True)
         schedule.cancel_job(sched_job)
         if options.support_bundle:
-            resp = collect_upload_sb_to_nfs_server(
-                MOUNT_DIR, sb_identifier, max_sb=CORIO_CFG["max_no_of_sb"])
-            LOGGER.info("collect support bundles response: %s", resp)
-        # TODO: cleanup object files created
+            support_bundle_process(CORIO_CFG["sb_interval_mins"] * 60, sb_identifier)
```

Comment:
Please revert this change since we don't want to collect support bundle in loop again. A process started at line 183 should already be collecting the support bundle in given interval. Here we just want to collect single support bundle due to failure.